ttnn/cpp/ttnn/operations/experimental/tensor/bcast/bcast_pybind.hpp

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "bcast.hpp"
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+#include "ttnn/operations/experimental/tensor/bcast/bcast.hpp"
```

Comment:
Please split into .hpp/.cpp
ccl/all_gather is a good example, it shows how to use pybind_fwd.hpp to avoid mega expensive includes in the header.

Reply:
done